packages/runtime/runtime-definitions/src/summary.ts

```diff
@@ -123,10 +123,32 @@ export interface IGarbageCollectionSnapshotData {
 	deletedNodes: string[] | undefined;
 }
 
+/**
+ * @experimental - Can be deleted/changed at any time
+ * Contains the necessary information to allow DDSes to do incremental summaries
+ */
+export interface IIncrementalSummaryContext {
+	/**
+	 * The sequence number of the summary generated that will be sent to the server.
+	 */
+	wipSummarySequenceNumber: number;
```

Comment:
`summarySequenceNumber` might be a better name. wip is confusing and doesn't really tell anything.